tcms/urls.py

```diff
@@ -25,6 +25,7 @@
     re_path(r"^$", core_views.DashboardView.as_view(), name="core-views-index"),
     re_path(r"^xml-rpc/", RPCEntryPoint.as_view(protocol=XMLRPC_PROTOCOL)),
     re_path(r"^json-rpc/$", RPCEntryPoint.as_view(protocol=JSONRPC_PROTOCOL)),
+    re_path(r"^setup/$", core_views.SetupView.as_view(), name="setup"),
```

Comment:
let's change this to `init-db` to match the new text. Also update middleware name & view class name accordingly.